lib/vmpooler/providers/gce.rb

```diff
@@ -424,10 +424,8 @@ def destroy_vm(pool_name, vm_name)
 
           unless deleted
             debug_logger("trigger delete_instance #{vm_name}")
-            vm_hash = get_vm(pool_name, vm_name)
             result = connection.delete_instance(project, zone(pool_name), vm_name)
             wait_for_operation(project, pool_name, result, 10)
-            dns_teardown(vm_hash)
```

Comment:
Please make sure that when a vm is deleted that dns_teardown is still being done

Reply:
Yes that is now a method in the DNS base class and being called at https://github.com/puppetlabs/vmpooler/blob/1406acf46a097cece52a0a43cdd1d04f24d40d7e/lib/vmpooler/pool_manager.rb#L518